.eslintrc.js

```diff
@@ -1,7 +1,6 @@
 module.exports = {
-    extends: 'get-off-my-lawn-prettier',
+    extends: 'get-off-my-lawn',
     rules: {
-        complexity: 0,
         'react/prop-types': 0
```

Comment:
Note to self that this can be removed with the latest release of get-off-my-lawn.